### ISSUE TITLE
Cannot Approve Flash Plugin when using MP4 or Flashls playback in CEF

### ISSUE THREAD

**Issue by matthiasfostel** [clappr/clappr#1844]
**Browser**: Chromium Embedded Framework 76.1.9

**OS**: Windows 10, x64

**Clappr Version**: latest

**Steps to reproduce**: 

1. Download CEF Sample Client from Spotify builds.
http://opensource.spotify.com/cefbuilds/index.html
2. Install PAPPI version of Adobe Flash plug-in.
https://get.adobe.com/flashplayer/otherversions/
3. Run cefclient.exe with "--enable-system-flash" execution flag.
4. Load a Clappr-based video webpage using an MP4 or HLS feed into the cefclient.
https://jsfiddle.net/amashoshin/a9cgx7sn/

Did you try to reproduce this issue at http://cdn.clappr.io/

Yes. Issue exists there too.

The clappr-rtmp-plugin's Flash approval works fine in CEF.

**Comment by towerz** [comment]
@matthiasfostel thanks for reporting this. We weren't aware of this flash permission issues since Flash is being phased out from the mainstream browsers.

We'll check this behavior.

**Comment by matthiasfostel** [comment]
I may not knowledgeable enough to troubleshoot but I tried to decipher the error in CEF and found that it seemed to be related to the Javascript applying a Class to the Flash Object. So, I removed the class parameter under the attributes key in the Flash Playback area of the code and it was able to run Flashls in CEF.

I did this by removing line 13047:

        `class: 'clappr-flash-playback',`

It caused other problems, though, like level-selector plugin's bitrate values aren't filling but I'm just happy it works now in my scenario. I have not tested to see if this fixes the MP4 playback.

Hope this helps!
